roles/wisdom/wisdom_llm_load_test_multiplexed/vars/main/resources.yml

```diff
@@ -0,0 +1,4 @@
+---
+llm_load_test_config_template: 'templates/llm-load-test-config.json.j2'
```

Comment:
I don't know why I'm not using this path format relative to the role base directory, I should rewrite all the resource files to use it .....